mlrun/config.py

```diff
@@ -1126,7 +1124,7 @@ def get_model_monitoring_file_target_path(
         artifact_path: str = None,
         function_name: str = None,
         **kwargs,
-    ) -> typing.Union[str, list[str]]:
+    ) -> str:
```

Comment:
```suggestion
    ) -> typing.Union[str, list[str]]:
```
I do think we need to keep that docstring as long as we support the mlrun<1.6.3 bc (L1165). We can modify the `return:` documentation, explaining that it can return a list only for older versions.

Reply:
We not support it any more, the function returning only a single string